libs/build_nco.sh

```diff
@@ -89,6 +89,10 @@ export LIBS="${PNETCDF_LIBS:-} ${NETCDF_LIBS:-} ${HDF5_LIBS:-} ${EXTRA_LIBS:-}"
 [[ -d build ]] && rm -rf build
 mkdir -p build && cd build
 
+# Prevents duplicate symbols
+# See http://nco.sourceforge.net/build_hints.shtml
+CPPFLAGS="-DHAVE_NETCDF4_H"
```

Comment:
Not that we will likely support installing netcdf version 3, but if we did, do we check for the netcdf version with this:
```
❯❯❯ nc-config --version
netCDF 4.7.4
```

Reply:
There is never a need to install an older version of netCDF. All netcdf libraries are fully backward compatible. NetCDF-3 code will run exactly the same, and produce the exact same output, with the latest version of netCDF.

Reply:
Presently, there are applications in the production suite that rely on netCDF 3.6.3 and link with it.

Reply:
Sure, why not. Added a check for NetCDF4.